lib/stdlib/doc/src/qlc.xml

```diff
@@ -1247,6 +1247,11 @@ ets:match_spec_run(
           <anno>Options</anno>, erl_eval:new_bindings())</c>.</p>
         <p>This function is probably mainly useful when called from
           outside of Erlang, for example from a driver written in C.</p>
+        <note>
+          <p>Query handles created this way may have worse
+            performance on evaluation than when created directly via
```

Comment:
This phrasing seems a little bit strange to me. What about:

```suggestion
            performance than when created directly via
```

Reply:
Sure, sounds good. I committed your suggestion.